packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -198,7 +207,7 @@ export function FileDropzone({ options, children, readAs, onLoad, fileListRender
     <div className={styles.container}>
       <div data-testid="dropzone" {...getRootProps({ className: styles.dropzone })}>
         <input {...getInputProps()} />
-        {children ?? <FileDropzoneDefaultChildren primaryText={getPrimaryText(files, options)} />}
+        {children ?? <FileDropzoneDefaultChildren primaryText={primaryTextSupplier(files, options)} />}
```

Comment:
I think we should add a test case for covering a custom text supplier in `FileDropzone.test.tsx`.

Reply:
You're right. I added a simple test to make sure it actually renders with the text from a passed supplier. Let me know what you think @leventebalogh.